hardware_interface/include/joint_limits_interface/joint_limits_interface.hpp

```diff
@@ -45,33 +46,33 @@ class JointLimitHandle
 {
 public:
   JointLimitHandle()
-  : prev_pos_(std::numeric_limits<double>::quiet_NaN()),
-    prev_vel_(0.0),
-    jposh_("position"),
+  : jposh_("position"),
     jvelh_("velocity"),
-    jcmdh_("position_command")
+    jcmdh_("position_command"),
```

Comment:
position_command was the name for (queue music) position command interface handles in previous code. So if you were to register a position state and a position command interface it would be "position" and "position_command". Has this changed in components?

Reply:
AFAIK, yes, so we would use "position" only now. I have to check this because `joint_limit_interface` works on the opposite since from controllers.